workitem/link/link_repository.go

```diff
@@ -97,6 +97,31 @@ func (r *GormWorkItemLinkRepository) ValidateCorrectSourceAndTargetType(ctx cont
 	return nil
 }
 
+// CheckParentExist returns error if there is an attempt to create more than 1 parent of a workitem.
+func (r *GormWorkItemLinkRepository) CheckParentExist(ctx context.Context, sourceID, targetID uint64, linkTypeID uuid.UUID) error {
```

Comment:
But in @DhritiShikhar 's defense, I did exactly the same with the Validate... function above hers ;)